CMakeLists.txt

```diff
@@ -2882,12 +2882,24 @@ IF(XNNPACK_BUILD_TESTS)
   TARGET_LINK_LIBRARIES(qd8-f32-qc8w-gemm-minmax-test PRIVATE gemm-microkernel-tester hardware-config logging microkernels-all microparams-init)
   ADD_TEST(NAME qd8-f32-qc8w-gemm-minmax-test COMMAND qd8-f32-qc8w-gemm-minmax-test)
 
+  ADD_EXECUTABLE(qd8-f16-qb4w-gemm-minmax-test test/qd8-f16-qb4w-gemm-minmax.cc)
```

Comment:
Since these are new tests *and* fp16 is a bit weird, perhaps OK for scalar but can you compile for all the scripts/build-*.sh and try to run at least x86 and android-armv7/8. I would also check for Google CI - even on my fork I think it will run IIRC, else we can see it when we run the CI like for WASM and RISC-V.